tasks/HelpMeCarry.tex

```diff
@@ -0,0 +1,76 @@
+\section{Help Me Carry [Party Host]}
+The robot helps a guest to carry some luggage to his/her car.
+
+\subsection{Main Goal}
+Robot helps a guest to carry a bag to the car which is parked outside and comes back home.
+
+\subsection{Focus}
+It includes understanding pointing gesture, object identification and manipulation, navigation in known and unknown environments, person-following while carrying a luggage, and social navigation.
+
+\noindent\textbf{Reward:} 500 pts for carrying the bag to the car's location and coming back home.
+
+\subsection{Bonus rewards}
+\begin{enumerate}[nosep]
+	\item Find the bag by understanding the pointing behaviour (100 pts).
+	\item Avoid all the obstacles outdoor (400 pts).
+\end{enumerate}
+
+
+\subsection{Setup}
+\begin{itemize}
+	\item \textbf{Location}: The test takes place inside and outside the arena.
+
+	\item \textbf{Start Location}: The robot starts at a predefined location in the living room.
+
+	\item \textbf{Guest}: There is only one guest in the house, standing in front of the robot at the start.
+	
+	\item \textbf{Bags' Location:} There are at least two bags placed between one to two meters  away from the guest's start position and both should be visible to the robot.
+	
+	\item \textbf{Pointing to the Bag:} When the test starts, the guest points to his/her bag and requests the robot to carry it to the car.
+	
+	\item \textbf{Outdoor Obstacles:} The robot will face multiple obstacles outside. The obstacles may include a person, a group of people, a stationary object or a moving object.
+\end{itemize}
+
+\subsection{Additional rules and remarks}
+\begin{enumerate}[nosep]
+	\item \textbf{Deus ex Machina:} Score reduction for requesting human assistance is applied as follows.
+	\begin{itemize}
+		\item help in finding the bag (-100 pts).
+		\item help in carrying the bag (-100 pts).
+		\item help in tracking the guest (-200pts).
+		\item help in going back home (-100pts).
+	\end{itemize}
+
+	\item \textbf{Pointing to the Bag:} The guest keeps pointing to the bag and does not move. 
+
+	\item \textbf{Car Location:} There is no car outside. Instead, a fixed location is supposed as a car location outside the arena.
+
+	\item \textbf{Reaching the Car:} The robot can reach the car's location only by following the guest. After the robot reaches the car's location, the guest says "Stop following me".
```

Comment:
~guest~ → operator

~After the robot reaches the car's location, the guest says "Stop following me".~ It is up to the robot to instruct the operator. We don't hard-code follow-me since 2009

Reply:
Then alternatively,

Reaching the Car: The robot can reach the car's location only by following the operator. The robot should instruct the operator how to start and stop following.

Reply:
I don't think so. To keep consistency with previous rulebooks you don't state that here. Instead, we normally add an additional rule about instructing the robot (you can find it in any Follow-me test, Navigation test, or Help-me-carry). Important is that during instruction:
- Max 1 minute
- No marks allowed
- No touching the robot
- **TEAM DO NOT INTERACT WITH THE OPERATOR** (I can never stress this enough)

Reply:
In the last year's HelpMeCarry description it says,

Following the operator: When the robot signals that it is ready to start, the operator starts walking –in a natural way– towards the car. Upon arrival, the operator will indicate the robot when they have reached their destination as instructed by the robot (e.g. by saying “here is the car” or “stop following me”).

So a similar rule could say,

Reaching the Car: The robot can reach the car's location only by following the operator. When the robot signals that it is ready to start, the operator starts walking –in a natural way– towards the car. Upon arrival, the operator will indicate the robot when they have reached their destination as instructed by the robot (e.g. by saying “here is the car” or “stop following me”).

Reply:
Or

Following the Operator: The robot should indicate the operator when it is ready to follow. The operator walks naturally towards the car. _After reaching the car, the operator takes the bag back from the robot and thanks the robot._

It will appear as a more natural way of stop following me. Do you think it will work?

Reply:
Sounds fine to me.